sdk/formrecognizer/Azure.AI.FormRecognizer/src/autorest.md

```diff
@@ -6,8 +6,9 @@ Run `dotnet build /t:GenerateCode` to generate code.
 > see https://aka.ms/autorest
 
 ``` yaml
-input-file:
-    - https://github.com/Azure/azure-rest-api-specs/blob/788507c386197b1ba7878fa00fe30871b8b01f22/specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json
+tag: release_2_1_preview.2
+require:
+    - https://github.com/Azure/azure-rest-api-specs/blob/788507c386197b1ba7878fa00fe30871b8b01f22/specification/cognitiveservices/data-plane/FormRecognizer/readme.md
```

Comment:
oh cool, so to make sure I understand it looks at that `md` file, then identifies `preview/v2.1-preview.2/FormRecognizer.json` because of the tag you set ?
And the idea is to keep all of our transformations here in this local md file, right?

Reply:
Correct @maririos!

Reply:
Correct (x2).